package-lock.json

```diff
@@ -27107,7 +27107,7 @@
                 "@aws-sdk/client-sso-oidc": "^3.616.0",
                 "@aws-sdk/token-providers": "^3.744.0",
                 "@aws/language-server-runtimes": "^0.2.78",
-                "@aws/lsp-core": "^0.0.4",
+                "@aws/lsp-core": "^0.0.5",
```

Comment:
hmm, shouldn't this have been done by the release/merge to main earlier 🤔

Reply:
Hmm not sure, but I see in main it's still 0.0.4